src/css/override_components.scss

```diff
@@ -135,15 +135,19 @@ button,
   }
 } /* stylelint-enable */
 
+.usa-button-disabled {
+  color: $color-textblack;
+}
+
```

Comment:
This is the only thing I don't agree with. I don't believe that disabled buttons need to meet contrast guidelines. A disabled button should not look active, and contrast guidelines are there to assure that colors appear active.

This is not only personal belief, but a web standard. The W3C contrast guidelines explicitly exclude inactive UI:

> Incidental: Text or images of text **that are part of an inactive user interface component,** that are pure decoration, that are not visible to anyone, or that are part of a picture that contains significant other visual content, have no contrast requirement. (http://ux.stackexchange.com/questions/80881/do-disabled-buttons-still-need-to-be-contrast-compliant-for-accessibility)

I recommend that this button get `$color-white` text with `$color-mediumgray-trans` background. It's still a bit dark for my taste (I might even prefer $color-lightgray` bg), but gets the point across.

![screen shot 2017-01-11 at 8 58 40 am](https://cloud.githubusercontent.com/assets/11464021/21857968/3588fa64-d7dc-11e6-935c-079fe2edd467.png)